README.md

```diff
@@ -20,7 +20,7 @@ Extensions allow to use threading in the background instead of using multiple pr
  
  - [eio](https://pecl.php.net/package/eio)
  - [php-uv](https://github.com/bwoebi/php-uv)
- - [pthreads](https://github.com/krakjoe/pthreads)
+ - [amphp/parallel](https://github.com/amphp/parallel)
```

Comment:
This is about extensions rather than implementations, so I guess this should link to `ext-parallel` instead of `ext-pthreads`.

Reply:
ext-parallel isn't used here directly though... and even with amphp/parallel it is not necessary to use the extension...

Reply:
It's using the extension by default if it's available. Above it says:

> Extensions allow to use threading in the background instead of using multiple processes.

`amphp/parallel` isn't optional and is the default.

Reply:
```suggestion
 - [parallel](https://github.com/krakjoe/parallel)
```